README.md

```diff
@@ -249,6 +249,16 @@ If you set `disallowedTagsMode` to `escape`, the disallowed tags are escaped rat
 
 If you set `disallowedTagsMode` to `recursiveEscape`, the disallowed tags are escaped rather than discarded, and the same treatment is applied to all subtags, whether otherwise allowed or not.
 
+
+#### "What if I wan disallowed tags and any content they contain should discarded"
+
+If you set `disallowedTagsMode` to `completelyDiscard`, disallowed tags and any content they contain are discarded. Any subtags are still included, as long as those individual subtags are allowed.
```

Comment:
Move this down with the explanation of the other option for `disallowedTagsMode`, below the subheading you created there, rather than having it here as a quoted remark. I think perhaps you just didn't quite finish reorganizing.

Reply:
I did the changes